app/services/resource_service.rb

```diff
@@ -54,7 +54,7 @@ def unchanged_attributes?(record)
     return @resource.compareable_attributes == record.compareable_attributes if record.respond_to?(:compareable_attributes)
 
     # Fallback, wenn :compareable_attributes nicht beim Model definiert ist.
-    except_attributes = ["id", "created_at", "updated_at", "tag_list", "category_id", "region_id"]
+    except_attributes = ["id", "created_at", "updated_at", "tag_list", "category_id", "region_id", "visible"]
```

Comment:
nice, so if all data stays the same with `visible` ignored (in addition to the other attributes) a record is a duplicate one. good! so the visibility of a record is always in manual hand, right?!

Reply:
exact